dashboard/config/scripts/levels/Fish Tank 1-validated.level

```diff
@@ -75,7 +76,7 @@
                 </block>
               </value>
               <next>
-                <block type="gamelab_addBehaviorSimple" inline="true">
```

Comment:
intentional?

Reply:
Same here...don't know how this happened